recipes/jenkins.rb

```diff
@@ -249,8 +256,14 @@ def jenkins = Jenkins.getInstance()
       --domain #{server_name}
       --fullchain-file #{cert_path}
       --key-file #{key_path}
+      --reloadcmd /root/cert-update-hook.sh
```

Comment:
A side of opening the door to execute more commands on reloading, why not using `--reloadcmd     "service nginx force-reload"`?

Reply:
> A side of opening the door to execute more commands on reloading, why not using `--reloadcmd "service nginx force-reload"`?

I created the hook script intentionally to add an indirection layer allowing us to expand the restarted services (in fact this entire thing is self-plagarized/adapted from my the itamae scripts I use to restart varnish, hitch, and nginx when SSL certs are renewed in my homelab).

To change our acme.sh configuration we either have to use the `--force` flag which forces an unnecessary certificate renewal or grok and modify the acme.sh configuration directly. Infrequent forced renewals won't be a problem but they are rate limited and that's an issue we could run into in the future. I also want to avoid depending on the acme.sh internal configuration to make changes.